recipe/meta.yaml

```diff
@@ -1,40 +1,48 @@
 {% set name = "jupyterlab_pygments" %}
-{% set version = "0.1.2" %}
+{% set version = "0.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: cfcda0873626150932f438eccf0f8bf22bfa92345b814890ab360d666b254146
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_pygments-{{ version }}.tar.gz
+  sha256: 7405d7fde60819d905a9fa8ce89e4cd830e318cdad22a0030f7a901da705585d
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  skip: True # [py<37]
 
 requirements:
   host:
     - python
     - pip
+    - jupyter-packaging >=0.10,<2
   run:
     - python
     - pygments >=2.4.1,<3
+  run_constrained:
```

Comment:
how come this is run_constrained? it's in the build system section of pyproject.toml, so shouldn't it be in the host section..?

Reply:
As far as I can see, this is actually not needed at build time (because of using the pypi release).
The run_constrained is required though, because this package is an extension to jupyterlab and is not compatible with version 4.

Reply:
if this package is an extension to jupyterlab, wouldn't this be a run_constrained in jupyterlab, not the other way round? and jupyterlab would be a hard run requirement here? I guess jupyterlab_pygments can't run without jupyterlab

Reply:
The package is a plugin. As a plugin can be developed externally, the run_constrained in this package makes sense.
I think jupyterlab_pygments can't run without jupyterlab, but I am not 100% sure (some other plugins run with notebook as well). It seems to be a common pattern for this plugin ecosystem to not set a dependency.